apps/web/src/lib/analytics.ts

```diff
@@ -0,0 +1,30 @@
+import mixpanel from "mixpanel-browser";
+import { getBaseUrl } from "../utils/base-url";
+const isProduction = process.env.NODE_ENV === "production";
+
+if (isProduction) {
+  mixpanel.init(process.env.NEXT_PUBLIC_MIXPANEL_TOKEN as string, {
+    api_host: `${getBaseUrl()}/mp`,
+    debug: !isProduction,
+  });
+}
+
+export function track(event: string, properties?: Record<string, unknown>) {
+  if (isProduction) {
```

Comment:
I think it would be nice to test tracking on `localhost` or at least in `preview`/`dev` environment?

Maybe we could go with multiple projects as suggested [here](https://help.mixpanel.com/hc/en-us/articles/360001354886-Automatically-Separate-Development-Data)

Reply:
Yes, that's actually the case! I've setup two projects, one for preview and one for production. Will change the `isProduction` name to something more appropriate